survey-runner-queue/ebs.tf

```diff
@@ -0,0 +1,9 @@
+module "ebs_bckup" {
+  source = "github.com/kgorskowski/terraform/modules//tf_ebs_bckup"
```

Comment:
@MIslam92 We should make this `source = "github.com/kgorskowski/ebs_bckup?ref=b08b842cac687af3dda4ccfc342bc4c2c0c692e5"` so that we lock to a specific version of the module.